code/modules/antagonists/changeling/abilities/changeling.dm

```diff
@@ -2,6 +2,9 @@
 	var/datum/abilityHolder/changeling/O = src.get_ability_holder(/datum/abilityHolder/changeling)
 	if (O)
 		return
+	var/mob/living/L = src
+	if(istype(L))
+		L.blood_id = "bloodc"
```

Comment:
This change should make it possible to remove some dumb `ischangeling()` checks from the blood code. You might want to do it in this PR if you want.

Reply:
Unfortunately, I'm not sure what half of the `ischangeling()` stuff is used for, but if its fairly easy, I can give it a shot.